package.json

```diff
@@ -25,10 +25,10 @@
   },
   "peerDependencies": {
     "bedrock": "1.18.1 - 3.x",
-    "bedrock-ledger-node": "8.x - 10.x"
+    "bedrock-ledger-node": "^11.0.0"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
If we're not testing this with 10, should we bump to 12?

Reply:
I bumped it to 12 here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity-storage/pull/6/commits/4ed786192522b59c60fa5bdb1c3c5f6e1fb0f6c9